Source/UI/CardListCell.swift

```diff
@@ -82,7 +82,7 @@ public class CardListCell: UITableViewCell {
         schemeImageView.image = schemeIconLocation(scheme: scheme)?.rawValue.image(forClass: CardListCell.self)
     }
 
-    private func schemeIconLocation(scheme: Card.Scheme) -> Constants.Bundle.SchemeIcons? {
+    private func schemeIconLocation(scheme: Card.Scheme) -> Constants.Bundle.SchemeIcon? {
```

Comment:
to this can be extension to Constants.Bundle.SchemeIcon. It may be reused for header view and card view.